partcad/src/partcad/shape_config.py

```
@@ -52,3 +53,43 @@ def get_final_config(self) -> dict:
     def get_store_data(self) -> ShapeConfigStore:
         final_config = self.get_final_config()
         return ShapeConfigStore(final_config)
+
+    async def get_mcftt(self, property: str):
```

Ruff (0.16.1)

[error] Function argument `property` is shadowing a Python builtin

(A002)